src/features/game/types/megastore.ts

```diff
@@ -137,6 +137,146 @@ const EPIC_PHARAOH_ITEMS: SeasonalStoreItem[] = [
   },
 ];
 
+const BULL_RUN_ITEMS: SeasonalStoreItem[] = [
+  {
+    wearable: "Cowboy Hat",
+    cost: {
+      items: {
+        Horseshoe: 50,
+      },
+      sfl: 0,
+    },
+  },
+  {
+    wearable: "Cowgirl Skirt",
+    cost: {
+      items: {
+        Horseshoe: 50,
+      },
+      sfl: 0,
+    },
+  },
+  {
+    collectible: "Cow Scratcher",
+    cost: {
+      items: {
+        Scarab: 50,
```

Comment:
Should these be "Cow Skull"?

Reply:
@brian-man Yes, it is just a placeholder. We are working on the new artefact today 😄